### DISCUSSION TITLE
Problems with Dashy/Portainer/Docker

### DISCUSSION THREAD

**Discussion by tombrau** [Lissy93/dashy#1650] [Help and Support] [answered]
Hi All,

I am having problems running Dashy from a compose file in portainer. My system is a win10 machine running Docker Desktop and Portainer. the compose file is,
version: "3.8"
services:
  dashy:
    # To build from source, replace 'image: lissy93/dashy' with 'build: .'
    # build: .
    image: lissy93/dashy
    container_name: Dashy
    # Pass in your config file below, by specifying the path on your host machine
    volumes:
      # - /root/my-config.yml:/app/user-data/conf.yml
      #- D:\\Docker\\dashy\\public\\conf.yml:/app/public/conf.yml
      #- D:\Docker\dashy\public\conf.yml:/app/public/conf.yml
      #- \\D\\Docker\\dashy\\public\\conf.yml:/app/public/conf.yml
      #- //D//Docker//dashy//public//conf.yml:/app/public/conf.yml
      #- /D/Docker/dashy/public/conf.yml:/app/public/conf.yml
      - D:\\Docker\\dashy\\public\\conf.yml:/app/public/conf.yml
    ports:
      - 8295:8080
    # Set any environmental variables
    environment:
      - NODE_ENV=production
    # Specify your user ID and group ID. You can find this by running `id -u` and `id -g`
      - UID=1000
      - GID=1000
    # Specify restart policy
    restart: unless-stopped
    # Configure healthchecks
    healthcheck:
      test: ['CMD', 'node', '/app/services/healthcheck']
      interval: 1m30s
      timeout: 10s
      retries: 3
      start_period: 40s`
`
the config file is
`
appConfig:
  theme: colorful
  layout: auto
  iconSize: medium
  language: en
pageInfo:
  title: Home Lab
  description: Welcome
  footerText: ''
sections:
  - name: Starter Only
    icon: fas fa-server
    items:
      - title: Google
        description: Search
        url: https://google.com
`

it looks like its reading the compose file, but its not reading the conf.yml file. I have tried all sorts of path statements but nothing seems to work. No changes I make to the conf file are read. I just get the std "Dashy Welcome to your new dashboard!" screen.

hope some one can help with this.

cheers
tom

**Comment by Cjdavidson** [comment, marked as answer]
I Had the same problem.

Try. - your directory:/app/user-data

https://github.com/Lissy93/dashy/discussions/1529

**Reply by tombrau** [reply]
Great, that worked, thanks very much.
the line that works is
D:\\Docker\\dashy\\public\\conf.yml:/app/user-data/conf.yml
again thanks very much
cheers
tom
